bve-native/src/parse/mesh.rs

```diff
@@ -9,10 +9,10 @@ use bve_derive::c_interface;
 use std::ffi::CStr;
 use std::ptr::null;
 
-pub use mesh::BlendMode as Blend_Mode;
-pub use mesh::FileType as File_Type;
-pub use mesh::Glow as Mesh_Glow;
-pub use mesh::GlowAttenuationMode as Mesh_Glow_Attenuation_Mode;
+pub use mesh::BlendMode;
```

Comment:
Why were these types aliased? Why do they no longer need to be aliased?

Reply:
These type were already C compatible, so didn't need a manual wrapper. I was aliasing them to bring them into the naming convention of the C api. However cbindgen shat itself and couldn't find these types when aliased like this. I could newtype them (make a `#[repr(transparent)]` wrapper around them to bring it into line if you think it's worth it.